ecdsa/keygen/round_1.go

```diff
@@ -39,22 +56,15 @@ func (round *round1) Start() *tss.Error {
 	Pi := round.PartyID()
 	i := Pi.Index
 
-	// 1. calculate "partial" key share ui
-	ui := common.GetRandomPositiveInt(tss.EC().Params().N)
-
-	round.temp.ui = ui
-
-	// 2. compute the vss shares
-	ids := round.Parties().IDs().Keys()
-	vs, shares, err := vss.Create(round.Threshold(), ui, ids)
+	vs, shares, err := round.genShares()
+	// clean up all the sensitive data
+	runtime.GC()
```

Comment:
This may cause the whole process to block momentarily. 
Zeroizing values in Go is very tricky and not fully supported, and I don't think this actually guarantees that the shares are cleared from memory. For more information, see https://github.com/golang/go/issues/21865.